app/src/components/features/mocksV2/MockEditorIndex/Editor/Header/index.tsx

```diff
@@ -90,7 +91,30 @@ export const MockEditorHeader: React.FC<HeaderProps> = ({
               }}
             />
           </Tooltip>
-          {!location.pathname.includes("rules") && <RQBreadcrumb />}
+          {!location.pathname.includes("rules") &&
+            (mockType === MockType.API ? (
+              <RQBreadcrumb
+                defaultBreadcrumbs={[
+                  {
+                    pathname: "",
+                    label: "File Server > JSON file",
+                    disabled: true,
+                    isEditable: false,
+                  },
+                ]}
+              />
+            ) : (
+              <RQBreadcrumb
+                defaultBreadcrumbs={[
+                  {
+                    pathname: "",
+                    label: "File Server > JS/CSS file",
+                    disabled: true,
+                    isEditable: false,
+                  },
+                ]}
+              />
+            ))}
```

Comment:
Refer rule [routes](https://github.com/requestly/requestly/blob/b20ef0187b151dd03f0645b75e9fe4975cfef50d/app/src/features/rules/routes.tsx#L34) for consuming breadcrumb